tox.ini

```diff
@@ -143,11 +143,9 @@ commands =
     python {toxinidir}/scripts/pycodestyle_on_repo.py
     python {toxinidir}/scripts/run_pylint.py
 deps =
-    {[testing]deps}
+    {[testenv]deps}
     pycodestyle
     pylint >= 1.6.4
-setenv =
-    PYTHONPATH = {toxinidir}/_testing
```

Comment:
It is still used for some of the docs-related environments, and the datastore emulator one.  Looks like `appveyor.yml` is still jamming it on the path, too. :(